sentry/values.yaml

```diff
@@ -1165,6 +1168,8 @@ kafka:
   socketRequestMaxBytes: "50000000"
   provisioning:
     enabled: true
+    # Topic list is based on https://github.com/getsentry/snuba/blob/master/snuba/utils/streams/topics.py
```

Comment:
There are additional topics at https://github.com/getsentry/self-hosted/blob/master/install/create-kafka-topics.sh#L6

Reply:
Added that in https://github.com/sentry-kubernetes/charts/pull/954/commits/4ce08e49e3688d519964c39e511989eb8c024bd2